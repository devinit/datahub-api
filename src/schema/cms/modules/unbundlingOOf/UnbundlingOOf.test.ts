import {getUnbundlingOOfPageData} from '.';
import * as prettyFormat from 'pretty-format';

describe('Unbundling OOf module tests', () => {
    it('should return page data for  Unbundling OOf', async () => {
        const pageData: DH.IPage[] = await getUnbundlingOOfPageData();
        expect(pageData.length).toBeGreaterThan(0);
        expect(prettyFormat(pageData)).toMatchSnapshot();
    }, 10000);
});
