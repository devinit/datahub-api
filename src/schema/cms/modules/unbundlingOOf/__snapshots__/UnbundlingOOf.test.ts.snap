// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Unbundling OOf module tests should return page data for  Unbundling OOf 1`] = `
"Array [
  Object {
    \\"id\\": \\"Unbundling aid\\",
    \\"meta\\": \\"\\",
    \\"narrative\\": \\"Explore and compare funding priorities for other official flows (OOFs)\\",
    \\"title\\": \\"Unbundling aid\\",
  },
]"
`;
